index.html

```diff
@@ -1,21 +1,101 @@
+          width="26"
```

Comment:
Avoid having this amount of image code in your markup - could you have simply linked the `src` attribute of your img element to '/img/karma-logo.svg'?